<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddTaskForm</class>
 <widget class="QWidget" name="AddTaskForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tasks</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0" colspan="2">
    <widget class="QTextEdit" name="notes_text_edit"/>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QComboBox" name="task_dropdown"/>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QComboBox" name="project_dropdown"/>
   </item>
   <item row="4" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="start_button">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="favourite_button">
       <property name="text">
        <string>Favourite</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_button">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="1">
    <widget class="QTimeEdit" name="time_worked_line"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
